### PR TITLE
Fix file name casing for linux

### PR DESCRIPTION
**Pull request by mzlarsson**
File cannot be found on linux (due to case sensitivity). We should keep names to lowercase only to avoid this in the future (my bad o.o).

### CODE REVIEW

**Review by kulturkrock-admin — APPROVED** [review]

**Comment by AntonNeld** [comment]
Do I understand it correctly that the risk with mixed case filenames is that a Windows user refers to a file (e.g. while importing) with different case than the actual filename, which works on Windows but not Linux?

We have some mixed case filenames in screencrash-ui, but this risk is lower in that repo since I am the main contributor. So I'm thinking it's worth it to keep them for the readability.

**Comment by mzlarsson** [comment]
> Do I understand it correctly that the risk with mixed case filenames is that a Windows user refers to a file (e.g. while importing) with different case than the actual filename, which works on Windows but not Linux?
> 
> We have some mixed case filenames in screencrash-ui, but this risk is lower in that repo since I am the main contributor. So I'm thinking it's worth it to keep them for the readability.

TL;DR: Yes :)

So the main problem here was that when developing on Windows you do not get any error when there is a mismatch between your casing in the code and the filename - hence you might introduce errors without noticing them. That was the main reason for me suggesting to keep lowercase file names (to remove the risk with hard-to-detect cross-platform errors introduced by Windows users). If you develop on Linux, these errors wouldn't be missed in the first place (cause stuff would crash).

**Comment by AntonNeld** [comment]
Good, I'll keep them then!
